datascience/__init__.py

```diff
@@ -3,4 +3,5 @@
 from .tables import *
 from .formats import *
 from .maps import *
+from .predicates import *
```

Comment:
This means that all the functions in `predicates.py` will be in the global scope when we run `from datascience import *`.

Are we concerned about potential name conflicts since `predicates.py` has a lot of functions?